x-pack/legacy/plugins/siem/server/lib/timeline/routes/import_timelines_route.ts

```diff
@@ -23,26 +28,32 @@ import {
   isBulkError,
   isImportRegular,
   ImportTimelineResponse,
-  ImportTimelinesRequestParams,
   ImportTimelinesSchema,
   PromiseFromStreams,
 } from './utils/import_timelines';
 
 import { IRouter } from '../../../../../../../../src/core/server';
 import { TIMELINE_IMPORT_URL } from '../../../../common/constants';
 import { SetupPlugins } from '../../../plugin';
-import { importTimelinesPayloadSchema } from './schemas/import_timelines_schema';
+import { ImportTimelinesPayloadSchemaRt } from './schemas/import_timelines_schema';
 import { importRulesSchema } from '../../detection_engine/routes/schemas/response/import_rules_schema';
 import { LegacyServices } from '../../../types';
 
 import { Timeline } from '../saved_object';
 import { validate } from '../../detection_engine/routes/rules/validate';
 import { FrameworkRequest } from '../../framework';
-
+import { throwErrors } from '../../../../../../../plugins/case/common/api';
+import { buildRouteValidation } from './schemas/export_timelines_schema';
 const CHUNK_PARSED_OBJECT_SIZE = 10;
 
 const timelineLib = new Timeline();
 
+interface ImportTimelinesRequestBodySchema {
+  file: Readable & {
```

Comment:
FYI We've already got [HapiReadableStream](https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/types.ts#L89-L93) in detections, but both it and `ImportTimelinesRequestBodySchema` should be unnecessary once `buildRouteValidation` is refactored to infer type information from the schema.